content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

```diff
@@ -27,24 +27,24 @@ If you create a new clone of the repository, you won't lose any of your Git hist
   $ cd <em>REPOSITORY-NAME</em>
   ```
 5. To filter out the subfolder from the rest of the files in the repository, run [`git filter-repo`](https://github.com/newren/git-filter-repo), supplying this information:
-    - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
+   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
```

Comment:
```suggestion
   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
```

Reply:
I used 3 spaces per [GitHub Flavored Markdown Spec 5.2 ex 274](https://github.github.com/gfm/#example-274).

Reply:
My apologies, just looked again and realized your revision is correct; had it in my head I had already set it to 3 spaces, but I guess I missed that line. Thanks.